### PR TITLE
bump openshift/docker-distribution

### PR DESCRIPTION
**Pull request by coreydaley**
Bump openshift/docker-distribution to get the recent VirtualHostedStyle update

### CODE REVIEW

File: vendor/github.com/docker/distribution/registry/storage/driver/gcs/gcs.go [hunk 1]
```diff
@@ -426,7 +426,8 @@ func putContentsClose(wc *storage.Writer, contents []byte) error {
 	var nn int
 	var err error
 	for nn < size {
-		n, err := wc.Write(contents[nn:size])
+		var n int
+		n, err = wc.Write(contents[nn:size])
```
Comment (adambkaplan) [inline vendor/github.com/docker/distribution/registry/storage/driver/gcs/gcs.go:430]: +1 - pick up the fix @dmage found in CI!

**Comment by coreydaley** [comment]
/hold
Waiting on https://github.com/openshift/release/pull/8109

**Comment by coreydaley** [comment]
Waiting on https://github.com/openshift/release/pull/8224

**Pull request merged**